std_msgs/Header header
string coming_from
string coming_to
string message
bool stop_action